composer.json

```diff
@@ -15,7 +15,8 @@
     "require": {
         "php": ">=5.3.3",
```

Comment:
since 5.3 end of life has been in mid 2014, I think we should at least use 5.5 as a compromise though it's also reached it end of life already.
see: http://php.net/supported-versions.php